build.gradle

```diff
@@ -18,17 +18,13 @@
 buildscript {
   repositories {
     jcenter()
-  }
-
-  // TODO: is this the right way to incluse gradle plugins?
-  repositories {
-    repositories {
-      maven {
-        // For license plugin gradle-rat.
-        url 'http://dl.bintray.com/content/netflixoss/external-gradle-plugins/'
-        // For avro compiler plugin gradle-avro-plugin
-        url 'https://plugins.gradle.org/m2/'
-      }
+    maven {
+      // For license plugin gradle-rat.
+      url 'http://dl.bintray.com/content/netflixoss/external-gradle-plugins/'
+    }
+    maven {
+      // For avro compiler plugin gradle-avro-plugin
+      url 'https://plugins.gradle.org/m2/'
     }
   }
```

Comment:
Can you please briefly explain what is the problem of the previous approach, so that latter folks who look at this PR can get a rough idea?